### PR TITLE
Added french quotes

### PR DESCRIPTION
**Pull request by AntoninMayet**
Hoping I didn't mess anything up.

### CODE REVIEW

**Comment by Miodec** [comment]
Please add a comment with translations so I can verify their content.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4918414921

File: frontend/static/quotes/french.json [hunk 1]
```diff
@@ -835,5 +835,55 @@
       "length": 105,
       "id": 138
     }
+    {
+    "text": "C'est propre, la tragédie. C'est reposant, c'est sûr... Dans le drame, avec ces traîtres, avec ces méchants acharnés, cette innocence persécutée, ces vengeurs, ces terre-neuve, ces lueurs d'espoir, cela devient épouvantable de mourir, comme un accident. On aurait peut-être pu se sauver, le bon jeune homme aurait peut-être pu arriver à temps avec les gendarmes. Dans la tragédie, on est tranquille. D'abord, on est entre soi. On est tous innocents, en somme !",
```
Comment (AntoninMayet) [inline frontend/static/quotes/french.json:839]: "It's clean, tragedy is. It's restful, it's safe... In drama, with those traitors, those ruthless villains, that persecuted innocence, those avengers, those Newfoundland dogs, those glimpses of hope, it becomes dreadful to die, like an accident. Perhaps we could have escaped, the good young man might have arrived in time with the police. In tragedy, one is at peace. First of all, one is among oneself. We are all innocent, in a way!"

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4938460386

**Comment by Miodec** [comment]
Whoops. I'll fix the issues.

**Pull request merged**